common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -57,279 +60,175 @@ nav.sequence-nav {
     @include linear-gradient(right, $shadow, $transparent);
     position: absolute;
     top: 0;
-    right: 0;
+    right: ($baseline*2);
     width: 20px;
-    height: 46px;
+    height: 44px;
     background-color: transparent;
     pointer-events: none;
+    z-index: 40;
   }
 
   .sequence-list-wrapper {
     @extend %ui-depth2;
     @include linear-gradient(top, #ddd, #eee);
-    position: relative;
+    @include box-sizing(border-box);
+    display: block;
     border: 1px solid $gray-l3;
     height: 44px;
-    margin: 0 ($baseline*1.5);
     box-shadow: 0 1px 3px $shadow-l1 inset;
-  }
-
-  ol {
-    position: absolute;
-    top: 0;
-    left: 0;
-    @include box-sizing(border-box);
-    display: table;
-    height: 100%;
-    margin: 0;
-    padding: 0 ($baseline/2);
     width: 100%;
 
-    a {
-      @extend .block-link;
-    }
-
-    li {
-      display: table-cell;
-      min-width: 20px;
+    .sequence-list {
```

Comment:
Here's a screengrab of my local DOM...

![screen shot 2015-01-28 at 10 40 13 am](https://cloud.githubusercontent.com/assets/163763/5940622/1259ec18-a6da-11e4-9058-5108bbc9a916.png)